client/public/index.html

```diff
@@ -50,6 +50,34 @@
       name="description"
       content="The MDN Web Docs site provides information about Open Web technologies including HTML, CSS, and APIs for both Web sites and progressive web apps."
     />
+
+    <!--
+      Why these Open Graph meta tags? ...when the '<title>' and '<meta name=description>'
+      already exist?
+      Well, because of Twitter.
+      When you paste in an MDN URL into Twitter it will not make a "card".
+      See this comment: https://github.com/mdn/yari/issues/3590#issuecomment-825153396
+
+      According to Twitter's documentation on "Cards" they will use their proprietary
+      tags (e.g. `<meta name="twitter:card" content="summary">`) but if not available
+      they will fall back to Open Graph meta tags.
+      See https://developer.twitter.com/en/docs/twitter-for-websites/cards/guides/getting-started#opengraph
+
+      It is a lot of repetition and additional bloat to the index.html files. But
+      keep in mind that repeated strings of texts compresses very well and most
+      clients will only download the Gzip or Brotli compressed HTML file.
+
+      Remember, all of these are *default* values. The actual values are processed in
+      ssr/render.js and uses cheerio to replace the content on all of these.
+    -->
+    <meta property="og:url" content="https://developer.mozilla.org" />
+    <meta property="og:title" content="MDN Web Docs" />
+    <meta
+      property="og:description"
+      content="The MDN Web Docs site provides information about Open Web technologies including HTML, CSS, and APIs for both Web sites and progressive web apps."
+    />
+    <meta property="og:locale" content="en-US" />
+
```

Comment:
The `og:image` and `og:image:alt` default tags are missing to formulate a social share image for the card.

We could add the generic MDN Web Docs [vector image](https://hacks.mozilla.org/files/2020/10/mdn-web-docs-vector-logo.png) from mozilla hacks document metadata that looks really nice for now. Using `<meta name="og:image:alt">` tag would be nice too, but mozilla hacks pages don't currently use one.

```html
<meta property="og:image" content="https://hacks.mozilla.org/files/2020/10/mdn-web-docs-vector-logo.png">
```

If the [image](https://hacks.mozilla.org/files/2020/10/mdn-web-docs-vector-logo.png) exists somewhere in Yari, we could replace the `content` with a Yari URL like https://developer.mozilla.org.